tools/ci_build/github/apple/build_ios_framework.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import os
+import pathlib
+import json
+import subprocess
+import sys
+import glob
+import shutil
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+REPO_DIR = os.path.normpath(os.path.join(SCRIPT_DIR, "..", "..", "..", ".."))
+BUILD_PY = os.path.join(REPO_DIR, "tools", "ci_build", "build.py")
+
+# We by default will build below 2 archs
+DEFAULT_BUILD_OSX_ARCHS = [["iphonesimulator", "x86_64"], ["iphoneos", "arm64"]]
+
+
+def _parse_build_settings(args):
+    _setting_file = args.build_settings_file.resolve()
+
+    if not _setting_file.is_file():
+        raise FileNotFoundError('Build config file {} is not a file.'.format(_setting_file))
+
+    with open(_setting_file) as f:
+        _build_settings_data = json.load(f)
+
+    build_settings = {}
+
+    if 'build_osx_archs' in _build_settings_data:
+        build_settings['build_osx_archs'] = []
```

Comment:
>build_settings['build_osx_archs'] [](http://example.com/codeflow?start=8&length=33)

how about using the same format for _build_settings_data["build_osx_archs"] and DEFAULT_BUILD_OSX_ARCHS? then we could simply have:
`build_settings["build_osx_archs"] = _build_settings_data.get("build_osx_archs", DEFAULT_BUILD_OS_ARCHS)`